movie_REST_API/movie_REST_API/App/Controllers/InfoMovieViewController.swift

```diff
@@ -234,22 +205,20 @@ final class InfoMovieViewController: UIViewController {
 
     private func loadMoviesData() {
         actorView.fetchMoviesData(id: idNew) { [weak self] in
-            DispatchQueue.main.async {
-                self?.imageCollectionView.reloadData()
-            }
+            self?.imageCollectionView.reloadData()
         }
     }
-    
-    @objc func goToWebButtonAction(sender: UIButton) {
+
+    @objc private func goToWebButtonAction(sender: UIButton) {
         let wkWebVC = WKWebViewController()
         wkWebVC.id = idNew
         navigationController?.pushViewController(wkWebVC, animated: true)
```

Comment:
```suggestion
        navigationController?.pushViewController(wkWebVC, animated: true)
```
это в роутере должно быть